<!DOCTYPE html>
<!-- This site was created with Hugo Blox. https://hugoblox.com -->
<!-- Last Published: October 7, 2025 --><html lang="en-us" >


<head><script src="/Bader-Website/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=Bader-Website/livereload" data-no-instant defer></script>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Hugo Blox Builder 5.9.7" />
  

  
  












  
  










  







  
  
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
  

  
  
  
    
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media="print" onload="this.media='all'">
    
  

  
  

  
  

  

  <link rel="stylesheet" href="/Bader-Website/css/vendor-bundle.min.css" media="print" onload="this.media='all'">

  
  
  
    
    

    
    
    
    
      
      
    
    
    

    
    
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      
        
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  
  <link rel="stylesheet" href="/Bader-Website/css/wowchemy.css" />

  
  
  

  
  
  
  
  
  
  
    
    
    <link rel="stylesheet" href="/Bader-Website/css/libs/chroma/github-light.min.css" title="hl-light" media="print" onload="this.media='all'" disabled>
    <link rel="stylesheet" href="/Bader-Website/css/libs/chroma/dracula.min.css" title="hl-dark" media="print" onload="this.media='all'" >
  

  
  



























  
  
  






  <meta name="author" content="David A. Bader" />





  

<meta name="description" content="Distinguished Professor and Director of the Institute for Data Science" />



<link rel="alternate" hreflang="en-us" href="http://localhost:1313/Bader-Website/blog/" />
<link rel="canonical" href="http://localhost:1313/Bader-Website/blog/" />





<link rel="icon" type="image/png" href="/Bader-Website/media/icon_hu_17e9a6122551a9eb.png" />
<link rel="apple-touch-icon" type="image/png" href="/Bader-Website/media/icon_hu_b914b33f8e933fd7.png" />

<meta name="theme-color" content="#bbdefb" />










  






<meta property="twitter:card" content="summary_large_image" />
<meta property="twitter:image" content="http://localhost:1313/Bader-Website/media/sharing.jpg" />



  

<meta property="og:type" content="website" />
<meta property="og:site_name" content="David A. Bader" />
<meta property="og:url" content="http://localhost:1313/Bader-Website/blog/" />
<meta property="og:title" content="Posts | David A. Bader" />
<meta property="og:description" content="Distinguished Professor and Director of the Institute for Data Science" /><meta property="og:image" content="http://localhost:1313/Bader-Website/media/sharing.jpg" /><meta property="og:locale" content="en-us" />

  
    <meta property="og:updated_time" content="2025-09-24T10:20:41-04:00" />
  










  
  
  

  
  
    <link rel="alternate" href="/Bader-Website/blog/index.xml" type="application/rss+xml" title="David A. Bader" />
  

  


  
  <title>Posts | David A. Bader</title>

  
  
  
  











</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="page-wrapper  dark " data-wc-page-id="b0d8ccffb8d530bff08365a148a04ee6" >

  
  
  
  
  
  
  
  
  
  <script src="/Bader-Website/js/wowchemy-init.min.js"></script>

  




  <div class="page-header header--fixed">
  
  
  
  
  












<header>
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/Bader-Website/">David A. Bader</a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/Bader-Website/">David A. Bader</a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/Bader-Website/#about"><span>Home</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/Bader-Website/#posts"><span>Posts</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/Bader-Website/#projects"><span>Projects</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/Bader-Website/#talks"><span>Talks</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/Bader-Website/publication/"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/Bader-Website/#contact"><span>Contact</span></a>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
            
            <li class="nav-item d-none d-lg-inline-flex">
              <a class="nav-link" href="https://www.facebook.com/ProfDavidBader" data-toggle="tooltip" data-placement="bottom" title="Follow me on Facebook" target="_blank" rel="noopener" aria-label="Follow me on Facebook">
                <i class="fab fa-facebook" aria-hidden="true"></i>
              </a>
            </li>
          
            
            <li class="nav-item d-none d-lg-inline-flex">
              <a class="nav-link" href="https://twitter.com/Prof_DavidBader" data-toggle="tooltip" data-placement="bottom" title="Follow me on Twitter" target="_blank" rel="noopener" aria-label="Follow me on Twitter">
                <i class="fab fa-twitter" aria-hidden="true"></i>
              </a>
            </li>
          
        

        
        
        

        
        
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    
    
    

    













  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1>Posts</h1>

  

  
</div>



<div class="universal-wrapper">
  


  

  
  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091216-isgtw/" >GRAPPAling with evolutionary history</a>
    </div>

    
    <a href="/Bader-Website/blog/20091216-isgtw/"  class="summary-link">
      <div class="article-style">
        <p><em>By Miriam Boon, iSGTW</em></p>


















<figure  id="figure-this-figure-illustrates-how-gene-order-changes-among-the-eight-species-each-thin-line-represents-a-single-gene-and-its-position-in-the-different-species-most-genes-are-conserved-on-the-same-chromosomal-arm-or-muller-element-but-gene-order-is-shuffled-between-species-this-figure-appeared-in-the-july-2008-issue-of-genetics-image-courtesy-of-stephen-schaeffer">
  <div class="d-flex justify-content-center">
    <div class="w-100" ><img alt="This figure illustrates how gene order changes among the eight species. Each thin line represents a single gene and its position in the different species. Most genes are conserved on the same chromosomal arm or Muller element, but gene order is shuffled between species. This figure appeared in the July 2008 issue of Genetics. *Image courtesy of Stephen Schaeffer.*" srcset="
               /Bader-Website/blog/20091216-isgtw/NewIndex_front_hu_2cfb13c85df9324d.webp 400w,
               /Bader-Website/blog/20091216-isgtw/NewIndex_front_hu_8cae099c3e323050.webp 760w,
               /Bader-Website/blog/20091216-isgtw/NewIndex_front_hu_f4596ec42f8d4f79.webp 1200w"
               src="/Bader-Website/blog/20091216-isgtw/NewIndex_front_hu_2cfb13c85df9324d.webp"
               width="300"
               height="192"
               loading="lazy" data-zoomable /></div>
  </div><figcaption>
      This figure illustrates how gene order changes among the eight species. Each thin line represents a single gene and its position in the different species. Most genes are conserved on the same chromosomal arm or Muller element, but gene order is shuffled between species. This figure appeared in the July 2008 issue of Genetics. <em>Image courtesy of Stephen Schaeffer.</em>
    </figcaption></figure>

<p>We’ve known for several years
now that chimpanzees share 96
percent of our DNA. Our
technology tells us how closely
humans and chimps are related.
But it doesn’t tell us how we’re
related. We need new technology
for that.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 16, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091216-isgtw/" >
        <img src="/Bader-Website/blog/20091216-isgtw/featured_hu_747b2fff2041d70a.webp" height="88" width="150"
            alt="GRAPPAling with evolutionary history" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091214-ieee-fellow/" >Five Georgia Tech Faculty Members Elected as IEEE Fellows</a>
    </div>

    
    <a href="/Bader-Website/blog/20091214-ieee-fellow/"  class="summary-link">
      <div class="article-style">
        <p>The Georgia Institute of Technology is one of only six U.S. universities to have five of its faculty members elevated to the rank of IEEE Fellow, the most at any academic institution in the United States. The five Georgia Tech faculty members promoted to IEEE Fellow, effective January 1, 2010, are <strong>David A. Bader</strong>, Ian T. Ferguson, Richard A. Hartlein, David C. Keezer, and Emmanouil M. Tentzeris.</p>
<p>Chosen by the <a href="https://www.ieee.org/" target="_blank" rel="noopener">IEEE</a> board of directors, the IEEE Fellows class of 2010 consists of 309 engineering professionals from around the world. IEEE is the world&rsquo;s leading professional association for the advancement of technology, and the IEEE grade of Fellow is conferred by the board of directors upon a person with an extraordinary record of accomplishments in any of the IEEE fields of interest.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 14, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    
    <div class="btn-links">
      








  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="/Bader-Website/blog/20091214-ieee-fellow/20091214-IEEE-Fellow.pdf" target="_blank" rel="noopener">
  PDF
</a>

















    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091214-ieee-fellow/" >
        <img src="/Bader-Website/blog/20091214-ieee-fellow/featured_hu_c9607f47f611e417.webp" height="44" width="150"
            alt="Five Georgia Tech Faculty Members Elected as IEEE Fellows" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091208-abcnews/" >Why Does the Air Force Want Thousands of PlayStations?</a>
    </div>

    
    <a href="/Bader-Website/blog/20091208-abcnews/"  class="summary-link">
      <div class="article-style">
        <p>By KI MAE HEUSSNER</p>
<p>Guess what&rsquo;s on the U.S. Air Force&rsquo;s wish list this holiday season.</p>
<p>Sony&rsquo;s popular PlayStation 3 gaming console. Thousands of them.</p>
<p>The Air Force Research Laboratory in Rome, N.Y., recently issued a request for proposal indicating its
intention to purchase 2,200 PlayStation 3 (PS3) consoles.</p>
<p>But the military researchers don&rsquo;t plan to play &ldquo;Call of Duty: Modern Warfare 2&rdquo; or any of the season&rsquo;s
other blockbuster games. They plan to string the consoles together into a massive supercomputer and
study how well they can enhance the military&rsquo;s high-performance computing systems.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 8, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091208-abcnews/" >
        <img src="/Bader-Website/blog/20091208-abcnews/featured_hu_fc571353f313bdc6.webp" height="49" width="150"
            alt="Why Does the Air Force Want Thousands of PlayStations?" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091202-ieee-cs-fellow/" >CS Members Elevated to Fellows</a>
    </div>

    
    <a href="/Bader-Website/blog/20091202-ieee-cs-fellow/"  class="summary-link">
      <div class="article-style">
        <p>Seventy IEEE Computer Society members will be elevated to IEEE Fellow grade in 2010. The grade of Fellow recognizes unusual distinction in the profession.</p>
<p>The IEEE Board of Directors elevated 309 members to Fellow status. Computer Society members recommended for Fellow status in 2010 include:</p>
<ol>
<li>Tinku Acharya</li>
<li>Raj Acharya</li>
<li>Charu Aggarwal</li>
<li>Srinivas Aluru</li>
<li><strong>David Bader</strong></li>
<li>Grady Booch</li>
<li>Athman Bouguettaya</li>
<li>Lionel Briand</li>
<li>Douglas Burger</li>
<li>Srimat Chakradhar</li>
<li>Stefano Chiaverini</li>
<li>Thomas Cloonan</li>
<li>Laurent Cohen</li>
<li>Ray Dolby</li>
<li>Ahmed El-Magarmid</li>
<li>Elmootazbellah Elnozahy</li>
<li>Mário Figueiredo</li>
<li>William Gropp</li>
<li>Baining Guo</li>
<li>Richard Hartley</li>
<li>Yutaka Hata</li>
<li>Joseph Hellerstein</li>
<li>James Hendler</li>
<li>John Impagliazzo</li>
<li>Yannis Ioannidis</li>
<li>Dimitrios Ioannou</li>
<li>David Kaeli</li>
<li>Andrew Kahng</li>
<li>Matti Karjalainen</li>
<li>Nikola Kasabov</li>
<li>David Keezer</li>
<li>Fanny Klett</li>
<li>Andrew Laine</li>
<li>Kwei-Jay Lin</li>
<li>Chih-Min (Jimmy) Lin</li>
<li>John C.S. Lui</li>
<li>Kevin Lynch</li>
<li>Margaret Martonosi</li>
<li>Peter Marwedel</li>
<li>Peter Maxwell</li>
<li>Mark Maybury</li>
<li>Dejan Milojicic</li>
<li>Joseph Mitola</li>
<li>Prasant Mohapatra</li>
<li>Eliot Moss</li>
<li>Robin Murphy</li>
<li>Ashwini Nanda</li>
<li>Lynne Parker</li>
<li>Larry Peterson</li>
<li>Jonathon Phillips</li>
<li>Keshav Pingali</li>
<li>Chunming Qiao</li>
<li>Long Quan</li>
<li>Anand Raghunathan</li>
<li>Al Reddy</li>
<li>Michael Scott</li>
<li>Timoleon Sellis</li>
<li>Malcolm Slaney</li>
<li>Asim Smailagic</li>
<li>Aravind Srinivasan</li>
<li>Heinrich Stuttgen</li>
<li>Jie Tian</li>
<li>Nian-Feng Tzeng</li>
<li>Nitin Vaidya</li>
<li>Amitabh Varshney</li>
<li>Yi-Min Wang</li>
<li>Shumpei Yamazaki</li>
<li>Qing Yang</li>
<li>Feng Zhao</li>
<li>Xinhua Zhuang</li>
</ol>
<p>The Board of Directors confers the title of Fellow upon a person of outstanding and extraordinary qualifications and experience in IEEE-designated fields, who has made important individual contributions to one or more of those fields. For more information online visit <a href="http://www.ieee.org/fellowprogram" target="_blank" rel="noopener">http://www.ieee.org/fellowprogram</a>. To view the full list of Computer Society Fellows, go to <a href="http://www.ieee.org/web/membership/fellows/Societies/COMP.html" target="_blank" rel="noopener">http://www.ieee.org/web/membership/fellows/Societies/COMP.html</a></p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 2, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091202-ieee-cs-fellow/" >
        <img src="/Bader-Website/blog/20091202-ieee-cs-fellow/featured_hu_9eef4a71c42229b1.webp" height="54" width="150"
            alt="CS Members Elevated to Fellows" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091119-register/" >Nuke labs show the future of hybrid computing</a>
    </div>

    
    <a href="/Bader-Website/blog/20091119-register/"  class="summary-link">
      <div class="article-style">
        <p><em>By Timothy Prickett Morgan</em></p>
<p>The Hybrid Multicore Consortium is on a mission that perhaps all of computing - on the desktop and in the data center - will one day embark on: making hybrid computing architectures as easy to program and use as monolithic platforms have been.</p>
<p>There is a growing consensus - but by no means a complete one - that the future of energy-efficient and yet powerful systems will be based on the coupling of general purpose, multicore CPUs with various kinds of co-processors that also have hundreds of cores to do specific kinds of accelerations needed by particular applications. The trouble with these hybrid computing architectures, which can bring a lot of flops to bear, is that even the smartest people in the world complain about how hard it is to program them.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 19, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091119-register/" >
        <img src="/Bader-Website/blog/20091119-register/featured_hu_849d5a199eb54acf.webp" height="72" width="150"
            alt="Nuke labs show the future of hybrid computing" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091118-genomeweb/" >NSF Awards $1M to Support Petascale Computational Tools for Genome Rearrangement Research</a>
    </div>

    
    <a href="/Bader-Website/blog/20091118-genomeweb/"  class="summary-link">
      <div class="article-style">
        <p>The National Science Foundation has funded three research teams under a four-year
$1 million project to help ensure that certain bioinformatics tools are compatible with
upcoming petascale computers.</p>
<p>NSF awarded the grants under the American Recovery and Reinvestment Act to support
the development of algorithms that will infer evolutionary relationships from genomic
rearrangement events — a task that could take &ldquo;centuries&rdquo; to analyze on today&rsquo;s fastest
parallel computers, according to the agency.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 18, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    
    <div class="btn-links">
      








  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="/Bader-Website/blog/20091118-genomeweb/20091118-GenomeWeb.pdf" target="_blank" rel="noopener">
  PDF
</a>

















    </div>
    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091118-genomeweb/" >
        <img src="/Bader-Website/blog/20091118-genomeweb/featured_hu_e8101a413fcee9a3.webp" height="46" width="150"
            alt="NSF Awards $1M to Support Petascale Computational Tools for Genome Rearrangement Research" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091117-petaapps/" >NSF Funds Genomics Petascale Project</a>
    </div>

    
    <a href="/Bader-Website/blog/20091117-petaapps/"  class="summary-link">
      <div class="article-style">
        <p><em>By Matthew Dublin</em></p>
<p>A team of researchers led by Georgia Tech&rsquo;s <strong>David Bader</strong> are poised to bring genomic
evolution to petascale heights. Equipped with a four-year $1M grant courtesy of the
National Science Foundation&rsquo;s PetaApps program, Bader and his team are setting their
sights on developing algorithms to take advantage of petascale computing platforms for
studying genome rearrangement events. Bader says that they will be writing their own
open source software development framework from scratch and eventually releasing it
through the GNU GPU license on Sourceforge. The team says they will start off the peta
project by ramping up GRAPPA, an open-source application that reconstructs the
evolutionary relatedness among species first released by Bader in 2000, to take
advantage of petascale computers. &ldquo;GRAPPA is currently the most accurate method for
determining genome rearrangement, but it has only been applied to small genomes with
simple events because of the limitation of the algorithms and the lack of computational
power,&rdquo; stated Bader in an announcement. The teamed used the latest version GRAPPA
to analyze a dataset of a dozen bellflower genomes to determin the flower&rsquo;s evolutionary
relatedness one billion times faster than the original, non-parallelized version.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 17, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091117-petaapps/" >
        <img src="/Bader-Website/blog/20091117-petaapps/featured_hu_e8101a413fcee9a3.webp" height="46" width="150"
            alt="NSF Funds Genomics Petascale Project" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091117-petaapps-full/" >NSF Funds Petascale Algorithms for Genomic Relatedness Research</a>
    </div>

    
    <a href="/Bader-Website/blog/20091117-petaapps-full/"  class="summary-link">
      <div class="article-style">
        <p>Scientists at three universities will use funding from the American Recovery and Reinvestment Act to develop computational biology tools that researchers will use with next-generation computers to study genomic evolution, according to Georgia Tech.</p>
<p>The $1 million grant from the National Science Foundation&rsquo;s PetaApps program, which funds development of computer technologies for petascale machines that can conduct trillions of calculations per second, will include Georgia Tech, the University of South Carolina, and Pennsylvania State University.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 17, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091117-petaapps-full/" >
        <img src="/Bader-Website/blog/20091117-petaapps-full/featured_hu_e8101a413fcee9a3.webp" height="46" width="150"
            alt="NSF Funds Petascale Algorithms for Genomic Relatedness Research" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091117-petascalegenomic/" >Petascale computing tools could provide deeper insight into genomic evolution</a>
    </div>

    
    <a href="/Bader-Website/blog/20091117-petascalegenomic/"  class="summary-link">
      <div class="article-style">
        <p>Technological advances in high-throughput DNA sequencing have opened up the possibility of determining how living things are related by analyzing the ways in which their genes have been rearranged on chromosomes. However, inferring such evolutionary relationships from rearrangement events is computationally intensive on even the most advanced computing systems available today.</p>
<p>Research recently funded by the American Recovery and Reinvestment Act of 2009 aims to develop computational tools that will utilize next-generation petascale computers to understand genomic evolution. The four-year $1 million project, supported by the National Science Foundation&rsquo;s <a href="https://www.nsf.gov/pubs/2007/nsf07559/nsf07559.htm" target="_blank" rel="noopener">PetaApps program</a>, was awarded to a team of universities that includes the Georgia Institute of Technology, the University of South Carolina and The Pennsylvania State University.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 17, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091117-petascalegenomic/" >
        <img src="/Bader-Website/blog/20091117-petascalegenomic/featured_hu_1d625fc181cb86b.webp" height="151" width="150"
            alt="Petascale computing tools could provide deeper insight into genomic evolution" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/Bader-Website/blog/20091111-sc09/" >Georgia Tech focuses on experimental systems and computational sciences at SC09</a>
    </div>

    
    <a href="/Bader-Website/blog/20091111-sc09/"  class="summary-link">
      <div class="article-style">
        <p>The Georgia Institute of Technology, an emerging leader in high-performance computing research and education, will be showcasing scientific research at the technical edge at next week&rsquo;s SC09, the international conference on high-performance computing, networking, storage and analysis scheduled for Nov. 14-20, 2009, at the Oregon Convention Center in Portland, Oregon. An SC09 best paper nomination for work in computational biology, a new GPU based experimental HPC system in the works, and expert presence across a range of hardware, software and application domains are just some of the ways Georgia Tech will feature multidisciplinary, cross-industry research efforts focusing on computational scientific discovery and sustainable high-performance computing (HPC).</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 11, 2009
  </span>
  

  

  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/Bader-Website/blog/20091111-sc09/" >
        <img src="/Bader-Website/blog/20091111-sc09/featured_hu_5968cf1630e77176.webp" height="50" width="150"
            alt="Georgia Tech focuses on experimental systems and computational sciences at SC09" loading="lazy">
      </a>
    
  </div>
</div>

  

  

  
<nav class="mt-1">
  <ul class="pagination justify-content-center">
    
    <li class="page-item"><a class="page-link" href="/Bader-Website/blog/page/39/">&laquo;</a></li>
    
    
    <li class="page-item"><a class="page-link" href="/Bader-Website/blog/page/41/">&raquo;</a></li>
    
  </ul>
</nav>


</div>
  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  












  
  
  
  
  













  





  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://hugoblox.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Hugo Blox Builder</a> — the free, <a href="https://github.com/HugoBlox/hugo-blox-builder" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  


<script src="/Bader-Website/js/vendor-bundle.min.js"></script>




  

  
  

  








































<script id="page-data" type="application/json">{"use_headroom":true}</script>


  <script src="/Bader-Website/js/wowchemy-headroom.js" type="module"></script>










<script src="/Bader-Website/en/js/wowchemy.min.js"></script>

























</body>
</html>
